packages/mineral-ui-tokens/tokens/generic/space.json

```diff
@@ -31,6 +33,9 @@
     { "name": "space_stack_md",   "value": "{!spacing_2x}" },
     { "name": "space_stack_lg",   "value": "{!spacing_3x}" },
     { "name": "space_stack_xl",   "value": "{!spacing_4x}" },
-    { "name": "space_stack_xxl",  "value": "{!spacing_8x}" }
+    { "name": "space_stack_xxl",  "value": "{!spacing_8x}" },
+
+    { "name": "space_none",       "value": "{!spacing_none}" },
+    { "name": "space_custom_sm",  "value": "{!spacing_1.5x}" }
```

Comment:
Where is this used? Let's come up with a better name than "custom" based on the use case.

Reply:
let's not use custom. where is this used?

Reply:
rename to space_inset_sm_plus